eng/ci/benchmarkconfigs/hellohttp-net9.yml

```diff
@@ -0,0 +1,38 @@
+imports:
```

Comment:
Does this need to be 1ES based pipeline like the rest?

Reply:
This file is not used by the ado pipeline, but by the benchmarking tool, crank.

Reply:
Updated it to use 1ES based templates.